MAUI/ImageEditor/getting-started.md

```diff
@@ -258,4 +260,40 @@ public partial class MainPage : ContentPage
 {% endhighlight %}
 {% endtabs %}
 
-N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-image-editor-examples/tree/master/GettingStarted)
\ No newline at end of file
+N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-image-editor-examples/tree/master/GettingStarted)
+
+## Get the image stream
+The `GetImageStream` method used to return the image in the form of a stream.

```

Comment:
`GetImageStream` => add link, it is already available

Reply:
The `GetImageStream` method is used to return the edited image in the form of a stream.

Reply:
Updated.